### PR TITLE
Fix issue about hangonman doesn't display cloud backgroud.

### PR DESCRIPTION
**Pull request by wuhengzhi**
"\_locales" will be skipped when package the apk since "\_*" belongs to
ANDROID_AAPT_IGNORE pattern, so remove "\_locales" to "locales".

BUG=https://crosswalk-project.org/jira/browse/XWALK-2223

### CODE REVIEW

**Comment by wuhengzhi** [comment]
@davidmaxwaterman Please help to review, thanks.

**Comment by davidmaxwaterman** [comment]
I don't think this change is correct. This app doesn't only target Android and for other platforms the '_locales' directory is required.
Actually, in my opinion, there is nothing wrong with this as it is, and the fix is to exclude the '_*' pattern from the ANDROID_AAPT_IGNORE pattern. IINM, the build process for this app actually implements that, as does the crosswalk-app-tool :

<https://crosswalk-project.org/jira/browse/XWALK-4666>

<https://github.com/crosswalk-project/crosswalk-app-tools/commit/bc8c19e4fd54e634ecc21a3073a9cfc6cd3cbf3b>

In summary:

1) use the build process to buld the app and you won't see any problem (if there's a problem with the build process, then please open an issue for that),
2) alternatively, implement a change to exclude the '_*' pattern in your own build process.

If you have no objection, I'll reject this PR.

Thanks,

Max.

**Comment by wuhengzhi** [comment]
Thanks a lot for your detail explanation. I will close this PR.
